packages/expo-modules-autolinking/scripts/ios/cocoapods/sandbox.rb

```diff
@@ -0,0 +1,65 @@
+# Overrides CocoaPods `Sandbox` class to patch podspecs on the fly
+# See: https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/sandbox.rb
+
+require 'json'
+
+module Pod
+  class Sandbox
+    private
+
+    _original_store_podspec = instance_method(:store_podspec)
+
+    public
+
+    define_method(:store_podspec) do |name, podspec, _external_source, json|
+
+      # Patch `React-Core.podspec` for clang to generate correct submodules for swift integration
+      if name == 'React-Core'
```

Comment:
Both names can be versioned in Expo Go, so we should rather check if it contains these names

Reply:
good catch! for Expo Go, i used a slight different approach which to generate patched podspec when versioning. the main reason is that we don't have the ABI43_0_0 prefix concept in `expo-modules-autolinking`.